docs/os-ref.md

```diff
@@ -458,21 +457,19 @@ This capability is configurable through the overloadable hook:
 function allowRecoverability(address token) public view returns (bool);
 ```
 
-The default implementation of `allowRecoverability()` is just to return true for all tokens, but your overload could choose to not allow certain tokens or even ETH.
+The default implementation of `allowRecoverability()` is just to return true for all tokens but your overload could choose to not allow certain tokens or even ETH.
 
 **Depositable proxies**
 
 AppProxies start off not being able to receive ETH through the native, gas-limited `.send()` and `.transfer()` methods.
 
-However, this can be explicitly enabled through:
+This can be explicitly enabled through this function when an app wants to allow itself (as the proxy instance) to recieve ETH from other contracts using `.send()` or `.transfer()`:
```

Comment:
I would reword this a little bit, due to the duplicate "this"

Reply:
Fixed in 4ad2ab5